internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -21,9 +21,8 @@ CREATE TABLE IF NOT EXISTS alarm_event_record (
     extensions JSONB, -- Additional data for extensibility
 
     -- O-RAN additional data to create AlarmEventNotification
-    resource_id UUID NOT NULL, -- Same as manager_cluster_id for caas alerts
-    resource_type_id UUID NOT NULL, -- Derived from manager_cluster_id
-    notification_event_type INT NOT NULL, -- Should be enum calculated from current alert
+    resource_id UUID, -- Same as manager_cluster_id for caas alerts
```

Comment:
Specifically removed `notification_event_type` from table as this can be calculated when we are ready to send notification to subscriber using `alarm_acknowledged` and `alarm_status`